docs/docs/getting-started.md

```diff
@@ -110,58 +68,12 @@ Then reference it from `helloworld.html`:
 
 Note that some browsers (Chrome, e.g.) will fail to load the file unless it's served via HTTP.
 
-### Offline Transform
-
-First install the [Babel](http://babeljs.io/) command-line tools (requires [npm](https://www.npmjs.com/)):
+## Using React with NPM / Bower
```

Comment:
npm is officially lowercase (and is so elsewhere in the guide).
Let’ change this to `Using React with npm or Bower`